providers/file_input.rb

```diff
@@ -28,6 +30,8 @@
               'state_file' => new_resource.name,
               'severity' => new_resource.severity,
               'facility' => new_resource.facility
-    notifies :restart, resources(:service => 'rsyslog')
+    notifies :restart, resources('service[rsyslog]')
   end
+
+  new_resource.updated_by_last_action(t.updated_by_last_action?)
```

Comment:
@jtimberman care to explain how the nothing notification prevents failures?  That one is new to me